src/writer/libtest.rs

```diff
@@ -479,6 +479,11 @@ impl<W: Debug + World, Out: io::Write> Libtest<W, Out> {
             Scenario::Step(step, ev) => self.expand_step_event(
                 feature, rule, scenario, &step, ev, retries, false, cli,
             ),
+            #[allow(clippy::print_stdout)]
+            Scenario::Log(msg) => {
+                print!("{msg}");
```

Comment:
This looks a little bit strange as `self.output` is not used. How can we comment it to make clear what's going on? What if `self.output` is not `io::stdout()`?

Reply:
@tyranron [`tracing_subscriber::fmt::TestWriter` also does it and has good explanation why](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/struct.TestWriter.html)